### PR TITLE
Add proper implementation of `CLEARFLAP` API command

### LINKED ISSUE #113: Implement the `CLEARFLAP` API command
The `CLEARFLAP` API command clears the flapping state of an interface.  It takes two arguments: A router name and an `ifindex` value representing the interface.

Flapping states have not yet been implemented in Zino 2, so this should be considered blocked until we do.

Refs to original code:
- `doClearFlap` in `server.tcl`
- `clearFlap` in `flaps.tcl`

### PR DESCRIPTION
**Pull request by lunkwill42**
## Scope and purpose

Fixes #113.  #311 added a dummy implementation, this completes it.

## Contributor Checklist

Every pull request should have this checklist filled out, no matter how small it is.
More information about contributing to Zino can be found in the
[README](https://github.com/Uninett/zino/blob/master/README.md#developing-zino).

* [x] Added a changelog fragment for [towncrier](https://github.com/Uninett/zino/blob/master/README.md#before-merging-a-pull-request)
* [x] Added/amended tests for new/changed code 
* [x] Added/changed documentation
* [x] Linted/formatted the code with black, ruff and isort, easiest by using [pre-commit](https://github.com/Uninett/zino/blob/master/README.md#code-style)
* [x] The first line of the commit message continues the sentence "If applied, this commit will ...", starts with a capital letter, does not end with punctuation and is 50 characters or less long. See https://cbea.ms/git-commit/
* [ ] If applicable: Created new issues if this PR does not fix the issue completely/there is further work to be done

### CODE REVIEW

**Comment by lunkwill42** [comment]
Once merged, #113 and #63 can be finally be closed.

File: tests/api/legacy_test.py [hunk 1]
```diff
@@ -782,10 +788,29 @@ async def test_it_should_respond_with_ok_but_not_implemented(self, authenticated
...
+
+        state = authenticated_protocol._state
+        state.devices.devices[router_name] = device_state
+
+        event = state.events.create_event(router_name, 1, PortStateEvent)
+        event.ifindex = 1
+        event.flapstate = FlapState.FLAPPING
+        state.events.commit(event)
+
+        state.events.commit(event)
```
Comment (johannaengland) [inline tests/api/legacy_test.py:803]: Why are you committing the event twice?

**Review by johannaengland — APPROVED** [review]
Just the one little comment

File: tests/api/legacy_test.py (hunk 1 above)
Comment (lunkwill42) [reply tests/api/legacy_test.py]: Yup, just an unnecessary dupe. Fixed.

**Pull request merged**